bundles/org.openhab.binding.icalendar/src/main/java/org/openhab/binding/icalendar/internal/logic/Event.java

```diff
@@ -23,41 +23,44 @@
  * A single event.
  *
  * @author Michael Wodniok - Initial contribution
- * @author Andrew Fiddian-Green - Added support for event description
+ * @author Andrew Fiddian-Green - Added support for event
```

Comment:
The word "description" in this line has been removed. Was this intentionally?

Reply:
no, readded it.